src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -325,33 +321,78 @@ export const NewProposal = ({ navigation, data, query }) => {
   );
 };
 
-const ImageSelector = ({ control, imageUri, item, onChange, selectImage, value }) => {
-  const { buttonTitle, infoText, label, name } = item;
+const ImageSelector = ({ control, field, item, selectImage }) => {
+  const [errorMessage, setErrorMessage] = useState();
 
-  useEffect(() => {
-    // the `!value` control is used for editing the proposal
-    // the proposal contains an image, it is for showing the preview to the users
-    if (!value) {
-      onChange(imageUri);
-    }
-  }, [imageUri]);
+  const { buttonTitle, infoText } = item;
+  const { name, onChange, value } = field;
 
   return (
     <>
-      <Input hidden label={label} control={control} name={name} value={value} />
+      <Input
+        {...item}
+        control={control}
+        errorMessage={errorMessage}
+        hidden
+        name={name}
+        validate={true}
```

Comment:
we like it short, so `validate` for passing `true` is enoufg, like `hidden` :)

Reply:
18e756843bc8ae851b92b31cf86be6945d2450ab